analyzers/tests/SonarAnalyzer.Test/Rules/SecureRandomSeedsShouldNotBePredictableTest.cs

```diff
@@ -29,7 +29,7 @@ public class SecureRandomSeedsShouldNotBePredictableTest
     private readonly VerifierBuilder builder = new VerifierBuilder()
         .AddAnalyzer(() => new CS.SymbolicExecutionRunner(AnalyzerConfiguration.AlwaysEnabled))
         .WithOnlyDiagnostics(ChecksCS.SecureRandomSeedsShouldNotBePredictable.S4347)
-        .AddReferences(NuGetMetadataReference.BouncyCastle());
+        .AddReferences(NuGetMetadataReference.BouncyCastleCryptography());
```

Comment:
`BouncyCastle` is apparently old and deprecated, and I was missing `new SecureRandom(generator, int)` overload for scenario#3